Assets/Scripts/Constellation.cs

```diff
@@ -15,9 +15,8 @@ public class Constellation : MonoBehaviour
 
     public Color highlightColor = Color.red;
     public Color neutralColor = Color.white;
-    public Color lineColor = Color.white;
-    public Material lineMaterial;
-    public float lineWidth = .035f;
```

Comment:
We might eventually need to make `lineWidth` a value that can be set from the inspector on the `ConstellationsController`.  This is mostly due to the odd sizing that I saw in the Vuforia scene.  Might be useful to set without changing the prefab.

Reply:
Might be useful I guess - I'll see about adding it back in.

Reply:
Added a public value to the controller as suggested - should work for rescaling the lines